Alvar Viana introduction blog post (#1105)
debezium/debezium.github.io: _posts/2025-02-01-hello-debezium-community.adoc

---
layout: post
title:  Hello Debezium Community!
date:   2025-02-01 12:00:00 +0100
tags: [ community, news ]
author: alvarvg
---

Hello Debezium Community! 

I am Alvar Viana and exactly one month ago I joined Red Hat and the Debezium team. I am a software engineer with almost 10 years of professional experience.

> In this years, I have always worked for product companies/proyects. Started as backend/embedded developer for a company that was acquired ABB.

Reviewer: ```suggestion
In my career,  I have always worked for product companies/projects. 
I started as a backend/embedded developer for a company, ABB, that was later acquired.
```

> A few years later I changed my life, changed cities, changed jobs and changed roles. Since them, I have been working as a Quality Engineer, at adidas and Sysdig. 

Reviewer: ```suggestion
A few years later, I changed my life, cities, jobs, and roles. Since then, I worked at Adidas and Sysdig as a Quality Engineer.
```

> As of today, I am happily joining Red Hat and debezium.  

Reviewer: ```suggestion
As of today, I am happily joining Red Hat and Debezium.  
```

> My contact with red hat has been little, although since I was a child I have looked at the company with admiration, since half a year ago I have been in different selection processes.

Reviewer: ```suggestion
I've had limited contact with Red Hat, although since I was a child, I have admired the company; since half a year ago, I have been in different selection processes.
```

> But shorter has been the time I know about Debezium. I have done some testing in the past to keep two databases synchronized, but for one reason or another, we put that testing aside. 

Reviewer: ```suggestion
I have previously worked with Debezium to create a PoC to keep two databases synchronized. Unfortunately, that effort never moved beyond a poC, but that experience and knowledge will now be useful.
```
